### PR TITLE
Fix stencil

### PR DESCRIPTION
**Pull request by maxammann**
Fixes the low-level stencil feature by upgrading wgpu: https://github.com/gfx-rs/wgpu/pull/3151

### CODE REVIEW

**Review by Drabble — APPROVED** [review]

**Comment by maxammann** [comment]
I think we will wait until the PR is merged and then switch to some wgpu commit.

**Comment by maxammann** [comment]
Merging now because it could take some time

**Pull request merged**
